src/components/recordingcreator/recordinghelper.js

```diff
@@ -98,14 +95,12 @@ function createRecording(apiClient, programId, isSeries) {
 }
 
 function sendToast(msg) {
-    import('toast').then(({ default: toast }) => {
-        toast(msg);
-    });
+    toast(msg);
 }
 
 function showMultiCancellationPrompt(serverId, programId, timerId, timerStatus, seriesTimerId) {
     return new Promise(function (resolve, reject) {
-        import('dialog').then(({ default: dialog }) => {
+        import('../dialog/dialog').then((dialog) => {
```

Comment:
_[not related to PR]_ I think it should be `dialog.show` down there https://github.com/jellyfin/jellyfin-web/blob/2c899c93dff42d7cefc1c2cbeb0e63eaf5d02b8b/src/components/recordingcreator/recordinghelper.js#L132

Reply:
fixed as suggested.